docs/core-blockchain-concepts/proof-of-work.md

```diff
@@ -2,80 +2,73 @@
+produce a given hash. Decred’s Proof-of-Work uses the BLAKE-256
 hashing function.
 
-After a miner successfully adds a new block to the chain, miners 
+After a miner successfully adds a new block to the chain, miners
 begin searching for the next block. They do this by computing a hash
 function which contains the block header from the last block (proving
-they haven't started searching before the last block was added), plus  
+they haven't started searching before the last block was added), plus
 a "nonce". The nonce is simply a single digit appended to the block
-header. Its purpose is simply to make the hash function output random. 
+header. Its purpose is simply to make the hash function output random.
 Remember, if we change one digit of the input, the hash function gives
 a completely new random number. The first miner to create a hash that
 does not exceed a certain value wins the right to create the next block.
 By finding the nonce that creates this hash, they have proved they have
-contributed a certain amount of work (computational power). 
+contributed a certain amount of work (computational power).
 
-If an attacker modified a block (e.g. to double spend coins), because 
+<!-- TODO s-ben: just made this up, needs review -->
+If an attacker modified a block (e.g. to double spend coins), because
 the hash includes the block header from the previous block, they must
-continue to create blocks that include the modified transactions. As 
+continue to create blocks that include the modified transactions. As
 soon as an honest miner checks the blockchain and sees the illegal transaction,
-it will revert the chain to the original split (just made this up, needs
-review). Since
-this requires, on average, as much hashing power as the entire network 
-expended between the time the original modified block was created and 
-the present time, PoW makes it very expensive to attack. 
-
+it will revert the chain to the original split. Since
+this requires, on average, as much hashing power as the entire network
+expended between the time the original modified block was created and
+the present time, PoW makes it very expensive to attack.
 
 ### Difficulty
 
+<!-- TODO jholdstock: The higher this number, the **higher** the difficulty. This needs re-working -->
 The difficulty refers to the the value a miner's hash must be below
 to find a block. The higher this number, the lower the difficulty, which
-is a measure of the average amount of computational power a miner must 
-spend to find the hash. 
-
-[NOTE: below is taken basically word for word from Bitcoin docs, and 
-need updating]
+is a measure of the average amount of computational power a miner must
+spend to find the hash.
 
 To keep the network producing blocks on average every 5 minutes, every
-X blocks, the network uses timestamps stored in each block header to 
+144 blocks the network uses timestamps stored in each block header to
```

Comment:
Confirmed the values in this section on Matrix: https://matrix.to/#/!HEeJkbPRpAqgAwhXWO:decred.org/$156707443113231treFA:decred.org?via=decred.org&via=matrix.org&via=zettaport.com